lib/openstax/salesforce/active_force.rb

```diff
@@ -7,10 +7,13 @@ class << self
     # to allow the Client to be successfully instantiated.
     alias_method :original_sfdc_client, :sfdc_client
     def sfdc_client
-      unless original_sfdc_client.is_a?(OpenStax::Salesforce::Client)
+      if original_sfdc_client.is_a?(OpenStax::Salesforce::Client)
```

Comment:
The logic here has been reversed from the original. I think the `original_sfdc_client` class that should be in the `if` check is something else here, it should be whatever class comes from ActiveForce. Or revert to the original code because see the next comment.